bublik/interfaces/api_v2/config.py

```diff
@@ -18,13 +18,31 @@
 from bublik.data.models import Config, ConfigTypes, GlobalConfigNames
 from bublik.data.serializers import ConfigSerializer
 
+from django_filters import rest_framework as filters
+from django.contrib.postgres.fields import JSONField
+
 
 logger = logging.getLogger('')
 
 
+class ConfigFilterSet(filters.FilterSet):
+    class Meta:
+        model = Config
+        fields = ['type', 'name', 'is_active', 'version']
+        filter_overrides = {
+            JSONField: {
+                'filter_class': filters.CharFilter,
+                'extra': lambda f: {
+                    'lookup_expr': 'icontains',
+                }
```

Comment:
The trailing comma is missing (will be fixed as a result of applying the linter).